changelog.txt

```diff
@@ -1,6 +1,176 @@
+* Fixes UnsupportedOperationException thrown in SpringResourceAccessor in Spring Boot 3 Native Image by @justin-tay in https://github.com/liquibase/liquibase/pull/3959
+* Do not repeat MDC logging. by @filipelautert in https://github.com/liquibase/liquibase/pull/4088
+* Implement Strict global configuration support to control whether non-empty author attribute is allowed or not by @MalloD12 in https://github.com/liquibase/liquibase/pull/4044
+* Read Snowflake views definitions with the Snowflake-specific query by @LonwoLonwo in https://github.com/liquibase/liquibase/pull/3794
+* Fixed #3745 error generation intType when using autoIncrement=true with H2 V2 by @quonas in https://github.com/liquibase/liquibase/pull/4008
+* log generated databasechangelogsql without erroneously incrementing the order executed by 2 (DAT-13680) by @StevenMassaro in https://github.com/liquibase/liquibase/pull/4097
+* Remove native executor name property (DAT-13580) by @abrackx in https://github.com/liquibase/liquibase/pull/4108
+
+
+* [PRO] DAT-12881 Implement runWithSpoolFile attribute and createSpool property to control use of Oracle spooling by @wwillard7800 in https://github.com/liquibase/liquibase-pro/pull/843
+* [PRO] DAT-12814: '--rollback-on-error' should return ERROR and return code 1 when update fails by @StevenMassaro in https://github.com/liquibase/liquibase-pro/pull/905
+* [PRO] DAT-13968 Added missing overwrite property by @wwillard7800 in https://github.com/liquibase/liquibase-pro/pull/895
+* [PRO] DAT-13667 Make sure exception from SQLCMD is propagated back to be displayed by @wwillard7800 in https://github.com/liquibase/liquibase-pro/pull/902
+* [PRO] DAT-14096 Handle existence of a file extension when creating spool/sql/log files by @wwillard7800 in https://github.com/liquibase/liquibase-pro/pull/913
+* [PRO] DAT-13994: strip leading slashes from contexts by @StevenMassaro in https://github.com/liquibase/liquibase-pro/pull/921
+
+
+## New Contributors
+* @maartenc made their first contribution in https://github.com/liquibase/liquibase/pull/3707
+* @amrasarfeiniel made their first contribution in https://github.com/liquibase/liquibase/pull/3927
+* @nwcm made their first contribution in https://github.com/liquibase/liquibase/pull/3842
+* @AlexGruebel made their first contribution in https://github.com/liquibase/liquibase/pull/3926
+* @ponziani made their first contribution in https://github.com/liquibase/liquibase/pull/3378
+* @barthel made their first contribution in https://github.com/liquibase/liquibase/pull/3903
+* @DCCSKrezovic made their first contribution in https://github.com/liquibase/liquibase/pull/4001
+* @justin-tay made their first contribution in https://github.com/liquibase/liquibase/pull/3959
+* @quonas made their first contribution in https://github.com/liquibase/liquibase/pull/4008
+
+
+
+
+**Full Changelog**: https://github.com/liquibase/liquibase/compare/v4.19.1...v4.20.0
```

Comment:
This full changelog here seems to be a copy & paste issue? The correct one is below.